Week2/alaa.js

```diff
@@ -0,0 +1,97 @@
+//1
+console.log('Hallo', 'wereld!'); // Dutch
+console.log('hola', 'mundo!'); //spanish
+console.log('مرحبا', 'العالم!'); //Arabic
+//2
+console.log("I'm awesome"); // should use two quotations instead of one
+//3
+var x;
+console.log('the value of my variable x will be: undefined');
+x = 7; // initialize the value of x
+console.log('the value of my variable x will be 7');
+console.log(x);
+//4
+var y;
+y = 'hyf';
+console.log('the value of y will be :hyf');
+console.log(y);
+var y = 'code';
+console.log('the value of y will be :code');
+console.log(y);
+//5
+var z = 7.25;
+console.log(z);
+var a = Math.floor(z);
+console.log(a);
+var max = Math.max(z, a);
+console.log(max);
+//6
+var myLibraries = [];
+console.log('the value of the the array is []');
+var myFavoriteAnimals = ['dogs', 'elephants', 'turtles', 'cats'];
+console.log(myFavoriteAnimals);
+var newLength = myFavoriteAnimals.push('baby pig');
+console.log(newLength);
+//7
+let myString = 'this is a test';
+console.log(myString);
+myString.length;
```

Comment:
You don't need this line. If you remember from our class today, this is just an expression and not a statement. The computer doesn't know what to do with this line except evaluate the length of `myString`. If you want, you can convert this to a statement like `const len = myString.length;` and then do `console.log(len);` in the next line 🙂